bin/react-scripts.js

```diff
@@ -5,14 +5,17 @@ var args = process.argv.slice(3);
 
 switch (script) {
 case 'build':
-case 'start':
 case 'eject':
+case 'start':
```

Comment:
Can you also add a `test` command to main `package.json` that would work with `--debug-template`? Please see how we switch paths in `config/paths.js`. This lets us iterate super quickly on the template without running `create-react-app` every time we want to see if the commands still work.

You can rename the existing `test` command to `e2e`.